RetailCoder.VBE/UI/Settings/GeneralSettings.xaml

```diff
@@ -215,6 +216,13 @@
                             </DataGridTemplateColumn>
                         </DataGrid.Columns>
                     </DataGrid>
+
+                    <Label Content="{Resx ResxName=Rubberduck.UI.RubberduckUI, Key=GeneralSettings_ExperimentalFeatures}" FontWeight="SemiBold" />
+                    <Label x:Name="SC_Warning" Margin="20,-5,0,0" Content="{Resx ResxName=Rubberduck.UI.RubberduckUI, Key=GeneralSettings_EnableSourceControlWarning}"
```

Comment:
I was trying to make the caption of the checkbox change when the checkbox was clicked.  `x:Name` is not needed and will be removed.